frontend/app/src/App.tsx

```diff
@@ -308,7 +308,7 @@ export class App extends PureComponent<Props, State> {
       hostHideSidebarNav: false,
       sidebarChevronDownshift: 0,
       pageLinkBaseUrl: "",
-      queryParams: "",
+      queryParams: null,
```

Comment:
Note: this change from `""` to `null` as a default value is so that we can more easily differentiate when the queryParams have been reset to empty vs being uninitialized at App startup. It makes the logic check in `App.getQueryString()` much easier to understand.